packages/validator/src/services/attestation.ts

```diff
@@ -52,7 +52,7 @@ export class AttestationService {
     // A validator should create and broadcast the attestation to the associated attestation subnet when either
     // (a) the validator has received a valid block from the expected block proposer for the assigned slot or
     // (b) one-third of the slot has transpired (SECONDS_PER_SLOT / 3 seconds after the start of slot) -- whichever comes first.
-    await Promise.race([sleep(this.clock.msToSlot(slot + 1 / 3), signal), this.waitForBlockSlot(slot)]);
+    await Promise.race([this.waitForBlockSlot(slot), sleep(this.clock.msToSlot(slot + 1 / 3), signal)]);
```

Comment:
Why is it necessary to change the order here?

Reply:
It was not a necessity. It was changed to align with the text of the spec, which was placed as a comment

Reply:
What do you mean the spec? Where does the spec specify the order of promises in javascript?